api_fhir_r4/mapping/actvityDefinitionMapping.py

```diff
@@ -0,0 +1,136 @@
+from medical.models import Service
+
+from api_fhir_r4.configurations import GeneralConfiguration
+
+
+class ServiceTypeMapping(object):
+    SYSTEM = f"{GeneralConfiguration.get_system_base_url()}CodeSystem/activity-definition-service-type.html"
+
+    fhir_service_type_coding = {
+        Service.TYPE_PREVENTATIVE: {
+            "system": SYSTEM,
+            "code": "P",
+            "display": "Preventive",
+        },
+        Service.TYPE_CURATIVE: {
+            "system": SYSTEM,
+            "code": "C",
+            "display": "Curative",
+        },
+    }
+
+
+class UseContextMapping(object):
+    SYSTEM = "http://terminology.hl7.org/CodeSystem/usage-context-type"
+
+    fhir_use_context_coding = {
+        "gender": {
+            "system": SYSTEM,
+            "code": "gender",
+            "display": "Gender",
+        },
+        "age": {
+            "system": SYSTEM,
+            "code": "age",
+            "display": "Age Range",
+        },
+        "venue": {
+            "system": SYSTEM,
+            "code": "venue",
+            "display": "Clinical Venue",
+        },
+        "workflow": {
+            "system": SYSTEM,
+            "code": "workflow",
+            "display": "Workflow Setting",
+        },
+    }
+
+
+class PatientCategoryMapping(object):
```

Comment:
This mapping has also defined by @sniedzielski in medicationMapping.py. Having the same URI, it should be defined only once.